test/integration/server-test.js

```diff
@@ -17,11 +17,12 @@ test('server create event with reftype = tag', t => {
   server(robotMock)
 
   t.is(robotMock.on.lastCall.arg, 'create')
+  // what is going on here?
   const handleCreateEvent = robotMock.on.lastCall.args[1]
```

Comment:
robotMock.on is the `robot.on` function called here

https://github.com/hoodiehq/first-timers-bot/blob/7bc48a829d17f4e994a0b3821a1a92894c27ba78/server.js#L7

`.lastCall` is a shortcut by `simple-mock` for `.calls[0]`. So `robotMock.on.lastCall.args[1]` returns the `handleCreateEvent` that gets passed as handler for the `'create'` event.

Does that make sense?

Reply:
oh ok, so it's the handleCreateEvent from the actual app, we don't have to mock or pass it in the test. 👍 
Lol I didn't mean to leave that there, I was going to ask you about that next week in person. I'm just leaving notes for myself on things I'm not 100% clear yet or I'm not able to explain in my own words 😃